### PR TITLE
Change label on vods from viewers to views

### PR DESCRIPTION
**Pull request by dfloss**
addresses issue #43 
Let me know if there is a better place for the isVod logic and I can change it.

### CODE REVIEW

File: src/util/is-vod.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+//utility function used to determine if a given service is a vod or a live stream
+const isVod = (service) => {
+  switch (service){
```
Comment (KenanY) [inline src/util/is-vod.js]: nit: add a space before the `{` here

Comment (KenanY) [inline src/util/is-vod.js]: nit: add a space before the `{` here

File: src/util/is-vod.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+//utility function used to determine if a given service is a vod or a live stream
```
Comment (KenanY) [inline src/util/is-vod.js]: Can you rewrite this to be in JSDoc style? Might look something like [this](https://github.com/MemeLabs/Rustla2/blob/2538f8b46f56ba57a25efae8d6fe63c5976407e3/src/util/is-valid-advanced-url.js#L1-L12).

File: src/util/is-vod.js (hunk 1 above)
Comment (dfloss) [reply src/util/is-vod.js]: fixed.

File: src/util/is-vod.js (hunk 2 above)
Comment (dfloss) [reply src/util/is-vod.js]: Definitely! I've added that to the latest commit, let me know if that works.

**Pull request closed without merging**